db/schema.rb

```diff
@@ -502,8 +502,8 @@
     t.integer "taxid", null: false
     t.bigint "wikipedia_id"
     t.string "title"
-    t.text "summary"
-    t.text "description"
+    t.text "summary", limit: 16_777_215
```

Comment:
Are these unrelated schema changes intended? Was the schema file out of date before?

Reply:
Sorry another weird mangling I will manually bring this back into line.